site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -251,24 +251,28 @@
       }
   }
 
+  let wasQueueEmpty = true;
```

Comment:
is this necessary?

Reply:
Actually we discussed yesterday this should be trigger on each joins instead of only join to empty queue so I just made another commit to do this which doesn't have the wasQueueEmpty anymore.

Reply:
Nvm actually I think I do need this to determine whether a student joined or left the queue so I know when to trigger the notification.